tests/integration/targets/activation/tasks/main.yml

```diff
@@ -172,14 +201,296 @@
         that:
           - _result.changed
 
+    - name: Copy rulebook activation
+      ansible.eda.rulebook_activation:
+        name: "{{ activation_name }}-copy"
+        copy_from: "{{ activation_name }}"
+        project_name: "{{ project_name }}"
```

Comment:
As it is now no other param than the new_name is used by the copy operation, so they should not be needed. Unless we want to honor what it is described in the documentation and update the copied activation with the other params provided.  In this case, it should have a different project_name, to be able to check that the copied activation already has the new project and not the original. 

As it is now, checking if it is has the same project is unnecessary, you would be just testing the backend when copying, which is out of the scope of this test.